test/client.test.js

```diff
@@ -2370,11 +2370,80 @@ describe('ACC Client', function () {
 
             // Now simulate reusing the local storage. We need a new client to make sure we do not reuse
             // the in-memory cache of the client.
+            let callLengthOld = storage.getItem.mock.calls.length;
```

Comment:
This variable does not seem to be used. Can you remove it?

Reply:
done